cmd/snap-seccomp/main.go

```diff
@@ -74,6 +74,11 @@ package main
 //#define PF_QIPCRTR AF_QIPCRTR
 //#endif				// AF_QIPCRTR
 //
+//#ifndef AF_XDP
+//#define AF_XDP 42
```

Comment:
There is a typo here - this should be 44

Reply:
Oops! https://github.com/snapcore/snapd/pull/12289